### ISSUE TITLE
onnx2ncnn not supported Cast and split op.

### ISSUE THREAD

**Issue by laobadao** [Tencent/ncnn#1109]
Because tensorflow is not supported, I convert TF Pb to onnx by tf2onnx (tensorflow2onnx), include BasicLSTMCell, and one BasicLSTMCell op split to multiple ops in onnx, like Cast Split Sigmoid Tanh and so on. but onnx2ncnn is not supported , Can you fix it? please.

Cast not supported yet!
to=7
Split not supported yet!
axis=1
Split not supported yet!
axis=1
Split not supported yet!
axis=1
Split not supported yet!
axis=1

**Comment by laobadao** [comment]
Cast not supported yet!
  # to=7
Split not supported yet!
  # axis=1
Split not supported yet!
  # axis=1
Split not supported yet!
  # axis=1
Split not supported yet!
  # axis=1

**Comment by hellocsi** [comment]
Have you tried onnx-simplier?
https://github.com/daquexian/onnx-simplifier

**Comment by nihui** [comment]
duplicate of https://github.com/Tencent/ncnn/issues/1108

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
